.stylelintrc.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
     // This rule can lead to less readable code
     // @see: https://github.com/stylelint/stylelint-config-recommended/issues/14
     'declaration-block-no-redundant-longhand-properties': null,
+    'color-hex-length': 'long',
```

Comment:
Since all colors are in genereated code (and we plan a revision of our Stylelint config anyway), I'm OK with this rule.

⚠️  **Just remember we must update our Sass exporter so it produces the colors in correct format.**

cc @crishpeen

Reply:
I have fixed this in https://github.com/lmc-eu/spirit-design-system/pull/1029/commits/c870602e7d62806844aed175751ef64e593ecede

Reply:
I would:
1. remove this line
2. update exporter so it shortens even with alpha channel (#ffffff00 -> #fff0)
3. test exporter it returns short version everywhere where possible

Reply:
Fixed.